README.adoc

```diff
@@ -78,6 +78,50 @@ You can also download it from the https://github.com/pgrange/bash_unit/releases[
 
 endif::[]
 
+== pre-commit hook
```

Comment:
I would add an extra `=` so that this becomes a sub-section of the installation section. Is it fair to say that running it with pre-commit hook is an alternative way of installing it?